modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/configuration/DLPreviewLimitConfigurationProvider.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.document.library.configuration;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+/**
+ * @author Marco Galluzzi
+ */
+@ProviderType
+public interface DLPreviewLimitConfigurationProvider {
+
+	public long getMaxFileSize(String scope, long scopePK);
+
+	public long getMaxFileSizeLimit(String scope, long scopePK);
+
+	public int getMaxNumberOfPages(String scope, long scopePK);
+
+	public int getMaxNumberOfPagesLimit(String scope, long scopePK);
+
+	public long getUnlimitedMaxFileSize();
+
+	public int getUnlimitedMaxNumberOfPages();
```

Comment:
The provider interface should expose configuration values. The "unlimited" are not part of the configuration, so the feel out of place here. Can you expose them as regular constants as it is done everywhere else?

Reply:
Done. Now I'm using constants exposed in the ConfigurationProvider interface. Still, these values are not part of the saved Configuration but they are used to enforce that introduced values are correct.